hdfs-sink/src/main/java/org/springframework/cloud/stream/module/hdfs/sink/HdfsSinkProperties.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.hdfs.sink;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import javax.validation.constraints.Pattern;
+
+/**
+ * Used to configure those Hdfs Sink module options that are related to connecting to Hdfs.
+ *
+ * @author Thomas Risberg
+ */
+@ConfigurationProperties(prefix = "spring.cloud.stream.module.hdfs.sink")
```

Comment:
IDK - I thought we wanted simple names as currently in XD, however this may be useful for composed modules. I'll check the others.